kornia/augmentation/container/augment.py

```diff
@@ -470,14 +470,14 @@ def forward(  # type: ignore[override]
 
         return outputs
 
-    def __call__(
+    def __call__(  # noqa: D417
```

Comment:
here there's also docu

Reply:
Same note that some but not all arguments are documented.